### PR TITLE
Improve 5.0 Release notes

### PR DESCRIPTION
**Pull request by riseshia**
### Summary
- Remove Duplicated explanation.
- Fix some typo.
- Attach PR, commit link that missed one.

Thanks for reading this. and Have a nice day XD

### CODE REVIEW

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Please refer to the [Changelog][railties] for detailed changes.
     ([Pull Request](https://github.com/rails/rails/pull/22173))
 
 *   Deprecated the tasks in the `rails` task namespace in favor of the `app` namespace.
-    (e.g. `rails:update` and `rails:template` tasks is renamed to `app:update` and `app:template`.)
+    (e.g. `rails:update` and `rails:template` is renamed to `app:update` and `app:template`.)
```
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: ..tasks are renamed to..

**Comment by vipulnsward** [comment]
Rest looks good. @riseshia thanks for the change.

r? @rafaelfranca

File: guides/source/5_0_release_notes.md (hunk 1 above)
Comment (riseshia) [inline guides/source/5_0_release_notes.md]: I think it's more natural... If you don't agree I will revert this :)

Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: We are referring to two entities, so `are` is apt here.

Comment (riseshia) [inline guides/source/5_0_release_notes.md]: ok, I will apply 👍

**Pull request merged**